File: DSA/Gautam_Shyamnani_590013078/Day4Question1.cpp
#include<iostream>
#include<vector>
using namespace std;
int main(){
    vector<int>arr={-3,-1,-5,-2};
    int min=arr[0];
    int n=arr.size();
    for(int i=0;i<n;i++){
        if(arr[i]<min){
            min=arr[i];
        }
    }
    cout<<min<<endl;
    return 0;
}

File: DSA/PranyaGupta_500126032/Day4Question1.py
class Solution:
    def MinimumElement(self, arr):
        minele = arr[0]
        for n in arr:
            minele = min(n, minele)
        return minele
def main():
    user_input = input("Enter elements of the array (with space in between): ")
    arr = list(map(int, user_input.strip().split()))
    sol = Solution()
    result = sol.MinimumElement(arr)
    print(result)
if __name__ == "__main__":
    main()